brainscore_core/plugin_management/conda_score.py

```diff
@@ -39,29 +39,29 @@ def score_in_env(self) -> 'subprocess.CompletedProcess[bytes]':
         return completed_process
 
     @staticmethod
-    def read_score(library_path: Path):
-        score_path = CondaScore._score_path(library_path)
+    def read_score(library_path: Path, env_name: str):
+        score_path = CondaScore._score_path(library_path, env_name)
         with open(score_path, 'rb') as f:
             score = pickle.load(f)
             os.remove(score_path)
             return score
 
     @staticmethod
-    def save_score(score: Score, library_path: Path):
-        score_path = CondaScore._score_path(library_path.parent)
+    def save_score(score: Score, library_path: Path, env_name: str):
+        score_path = CondaScore._score_path(library_path.parent, env_name)
         with open(score_path, 'wb') as f:
             pickle.dump(score, f, pickle.HIGHEST_PROTOCOL)
 
     @staticmethod
-    def _score_path(library_path: Path) -> Path:
+    def _score_path(library_path: Path, env_name: str) -> Path:
         """
         File path inside the given library for sub-process to write the score to, and for us to read back in.
         Since separate scoring runs typically occur in separately installed libraries,
         the score paths between them will be different and not conflict.
         There is still a remaining race condition when two scoring runs are started within the same library,
         which is currently not handled.
         """
-        return library_path.parent / 'conda_score.pkl'
+        return library_path.parent / f'conda_score_{env_name}.pkl'
```

Comment:
```suggestion
        return library_path.parent / f'conda_score--{env_name}.pkl'
```
nitpicky, but would prefer a different separator between prefix and suffix than within each part